include/options.h

```diff
@@ -55,6 +56,10 @@ struct CheckOptions {
   // deletion is triggered by a timer. This value must be larger than
   // flush_interval_ms.
   const int expiration_ms;
+
+  // Only the attributes in this set are used to caclculate cache key.
+  // Must not be empty.  If empty,  cache is disabled.
```

Comment:
Should remove "Must not be empty", since the next sentence implies that it can in fact be empty.